future-apps/payroll/contracts/Payroll.sol

```diff
@@ -64,15 +65,16 @@ contract Payroll is EtherTokenConstant, IForwarder, IsContract, AragonApp {
         uint64 endDate;
     }
 
-    IFeed public feed;
     Finance public finance;
+    IFeed public feed;
     address public denominationToken;
     uint64 public rateExpiryTime;
 
+    // Employees start at index 1, to allow us to use employees[0] to check for non-existent address
     uint256 public nextEmployee;
-    mapping(uint256 => Employee) private employees;     // employee ID -> employee
-    mapping(address => uint256) private employeeIds;    // employee address -> employee ID
-    mapping(address => bool) private allowedTokens;
+    mapping(address => uint256) internal employeeIds;    // employee address -> employee ID
+    mapping(uint256 => Employee) internal employees;     // employee ID -> employee
```

Comment:
Could we move `uint256 public nextEmployee` to be after this line (since it acts as its length)?

Reply:
Sure, will do it in the next payroll PR :)